cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -248,6 +248,24 @@ public Builder(String schedule) {
       this.endingActions = new ArrayList<>();
     }
 
+    public Builder setDefaultConfig(ETLBatchConfig config) {
```

Comment:
it is better to have a `builder(ETLBatchConfig config)` method to return a Builder with values in the config